axelor-human-resource/src/main/resources/views/Timesheet.xml

```diff
@@ -81,7 +81,6 @@
 			<menu title="Tools" icon="fa-wrench">
 				<item title="Lines Generation Wizard" showIf="statusSelect == 1" action="action-timesheet-record-default-generation-date,action-timesheet-attrs-set-log-time-title,action-timesheet-attrs-generate-lines-panel-hidden-false"/>
 				<item title="Generate Lines From Expected Planning" showIf="statusSelect == 1" action="save,action-timesheet-method-generate-lines-from-expected-planning" />
-				<item title="Generate Lines From Realise Planning Project / Task" showIf="statusSelect == 1" action="save,action-timesheet-method-generate-lines-from-realise-planning" />
```

Comment:
also, remove related action-method from view and its java methods if not used anywhere